scripts/parse-tsdocs.js

```diff
@@ -1,27 +1,53 @@
 const docgen = require('react-docgen-typescript');
 const fs = require('fs');
+const path = require('path');
```

Comment:
I'm not that familiar with whats going on in this file, can you explain the changes that you've made here?